custom/icds/tasks.py

```diff
@@ -30,3 +46,41 @@ def run_indicator(domain, user_id, indicator_class):
 
         for message in messages:
             send_sms(domain, usercase, phone_number, message)
+
+
+def get_awc_location_ids(domain):
+    return get_location_ids_with_location_type(domain, AWC_LOCATION_TYPE_CODE)
+
+
+def get_supervisor_location_ids(domain):
+    return get_location_ids_with_location_type(domain, SUPERVISOR_LOCATION_TYPE_CODE)
+
+
+def is_first_week_of_month():
+    day = datetime.utcnow().day
+    return day <= 7
+
+
+@periodic_task(
+    run_every=crontab(hour=3, minute=30, day_of_week='mon'),
+    queue=settings.CELERY_PERIODIC_QUEUE,
+    ignore_result=True
+)
+def run_weekly_indicators():
```

Comment:
i had it that way at first, but then realized it would require iterating through all the users again, so decided to stick with doing everything in one task